.github/workflows/notify-team-to-review-pr.yml

```
@@ -0,0 +1,24 @@
+    types: [opened]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  add-label:
+    if: contains(github.event.pull_request.labels.*.name, 'konflux-nudge') == false
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add review-requested label
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)